modules/oadp-api-details.adoc

```diff
@@ -0,0 +1,233 @@
+|===
+
+link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#SnapshotLocation[Complete schema definitions for the type `SnapshotLocation`].
+
+.ApplicationConfig
+[options="header"]
+|===
+|Property|Type|Description
+
+|`velero`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#VeleroConfig[VeleroConfig]
+|Defines the configuration for the Velero server.
+
+|`restic`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#ResticConfig[ResticConfig]
+|Defines the configuration for the Restic server.
+|===
+
+link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#ApplicationConfig[Complete schema definitions for the type `ApplicationConfig`].
+
+.VeleroConfig
+[options="header"]
+|===
+|Property|Type|Description
+
+|`featureFlags`
+|[] link:https://pkg.go.dev/builtin#string[string]
+|Defines the list of features to enable for the Velero instance.
+
+|`defaultPlugins`
+|[] link:https://pkg.go.dev/builtin#string[string]
+|Five types of default Velero plug-ins can be installed: `AWS`,`Azure`, `CSI`, `GCP`, and `OpenShift`.
+
+|`customPlugins`
+|[]link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#CustomPlugin[CustomPlugin]
+|Used for installation of custom Velero plug-ins.
+
+link:https://github.com/openshift/oadp-operator/blob/076f1495f6de65b0e96392cb6ef7cff66cf7defb/docs/config/plugins.md[Plug-ins customization] gives more details on working with default and custom plug-ins.
+
+|`restoreResourcesVersionPriority`
+|link:https://pkg.go.dev/builtin#string[string]
+|Represents a configmap that is created if defined for use in conjunction with the `EnableAPIGroupVersions` feature flag. Defining this field automatically adds `EnableAPIGroupVersions` to the Velero server feature flag.
+
+|`noDefaultBackupLocation`
+|link:https://pkg.go.dev/builtin#bool[bool]
+|To install Velero without a default backup storage location, you must set the `noDefaultBackupLocation` flag in order to confirm installation.
+
+|`podConfig`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#PodConfig[`PodConfig`]
+|Defies the configuration of the `Velero` pod.
+
+|`logLevel`
+|link:https://pkg.go.dev/builtin#string[string]
+|Velero server’s log level (use Debug for the most granular logging, leave unset for Velero default). Valid options are `trace`, `debug`, `info`, `warning`, `error`, `fatal`, and `panic`.
+|===
+
+link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#VeleroConfig[Complete schema definitions for the type `VeleroConfig`].
```

Comment:
link is broken. Please use a link without a specific version, like:
https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#VeleroConfig maybe